index.html

```diff
@@ -13,73 +13,219 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Work+Sans:wght@400;600;700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Sora&display=swap" rel="stylesheet">
 </head>
 <body>
-    <section id="main">
+    <!-- Nav -->
+    <nav>
+        <div class="top">
+            <i id="toggle" class="fa fa-bars" aria-hidden="true" id="toggle"></i>
+            <a id="logo" href="https://brickhack.io">
+                <img src="assets/bh8/logo.svg" alt="BrickHack 8 Logo" />
+            </a>
+        </div>
+        <div class="left navbar-items">
+            <a class="link" href="#about">About</a>
+            <a class="link" href="#faq">FAQ</a>
+            <a class="link" href="#contact">Contact</a>
+        </div>
+        <div class="right">
+            <!-- Login and Signup links will go here -->
+        </div>
+    </nav>
+    <div class="mobile-grayout"></div>
+
+    <!-- MLH Banner -->
+    <a id="mlh-trust-badge" href="https://mlh.io/seasons/2022/events?utm_source=na-hackathon&utm_medium=TrustBadge&utm_campaign=2022-season&utm_content=gray" target="_blank">
+        <img src="https://s3.amazonaws.com/logged-assets/trust-badge/2022/mlh-trust-badge-2022-gray.svg" alt="Major League Hacking 2022 Hackathon Season" style="width:100%">
+    </a>
+
+    <!-- Hero -->
+    <section id="hero">
+        <div id="hero-content">
+            <h1>BrickHack 8</h1>
+            <h3>March 5-6, 2022</h3>
+            <h3>RIT Online Hackathon</h3>
+            <p>Apply today: </p>
+            <!-- Login and Signup buttons -->
+            <div id="hero-buttons">
+                <a id="login" href="https://apply.brickhack.io/">LOGIN</a>
+                <a id="register" href="https://apply.brickhack.io/users/sign_up">REGISTER</a>
+            </div>
+        </div>
+        <div id="hero-video">
+            <video width="750" autoplay muted loop playsinline>
+                <source src="./assets/bh8/videos/LandingLoop.mp4" type="video/mp4">
+            </video>
+        </div>
+    </section>
 
-        <img id="logo" src="./resources/pixel-logo-real.png" alt="BrickHack 8 logo. Two stripes, left orange and right blue, in a parallelogram with a white border.">
+    <!-- About -->
+    <section id="about">
+        <div id="about-text">
+            <h2>What is a Hackathon?</h2>
+            <p>A hackathon is an invention marathon. BrickHack is a 24-hour digital hackathon that is completely free and open to any university student 18+ years old.</p>
+            <p>Apply to come spend March 5th and 6th, 2022 with us to learn, collaborate, build, and innovate!</p>
+        </div>
+        <div id="about-cards">
+            <div class="about-card">
+                <h3>Applying</h3>
+                <p>When applications open, you will apply to BrickHack via the HackEngine system. Applications will be open until the event opens on March 5th.</p>
```

Comment:
We should probably modify this to point out how to access HackEngine as we don't actually define how to do that. Or we should redo this section as applications will be open when site goes live? I think?

Reply:
Applications should be live when site goes live
Can we run this piece of content by marketing for a reword?

Reply:
Yeah! Want me to message Adam while you work w/ design on other tweaks (re: checklist above)?

Reply:
I will send something in #marketing about it. The checklist was pretty much already done (got sign offs from both teams). Just need to fix this and possibly your other questions/tweaks with design team